rust/feature-flags/src/api/request_handler.rs

```diff
@@ -688,15 +821,19 @@ mod tests {
 
         let feature_flag_list = FeatureFlagList { flags };
 
-        let evaluation_context = FeatureFlagEvaluationContextBuilder::default()
-            .team_id(1)
-            .distinct_id("user123".to_string())
-            .feature_flags(feature_flag_list)
-            .reader(reader)
-            .writer(writer)
-            .cohort_cache(cohort_cache)
-            .build()
-            .expect("Failed to build FeatureFlagEvaluationContext");
+        let evaluation_context = FeatureFlagEvaluationContext::new(
+            1,
+            1,
```

Comment:
Nit: Per my comment on strongly typed IDs, I try to have at least one test where we pass different values for IDs to ensure the test doesn't pass by luck.

For example, this might pass in both calls:

```rust
let evaluation_context = FeatureFlagEvaluationContext::new(
     team.id,
     team.project_id,
     ...
     
let evaluation_context = FeatureFlagEvaluationContext::new(
     team.project_id,
     team.id,
     ...
```

Which would hide a bug. I'm only paranoid about this because of the number of times it's happened to code I've worked on. Was it my fault or someone else's? I'll never tell!

Reply:
Yeah, that's a good call, let me see if I can come up with some way to wrap these data.